azure/azure_style.yaml

```diff
@@ -52,3 +52,17 @@ jobs:
           cd ${{ parameters.REPO_NAME }}
           pip install black==20.8b1
           black . --check --diff -l 120 --target-version py37 --target-version py38
+
+  - job: isort
+    pool:
+      vmImage: "ubuntu-20.04"
+    continueOnError: ${{ parameters.IGNORE_ISORT }}
```

Comment:
This will run on all pipelines and give them the 'partially succeeded' badge if they don't use `isort`. I think it would be better to just skip the job with `condition: not(${{ parameters.IGNORE_ISORT }})`

Reply:
Ah, excellent point.